src/Elastic.Apm/Api/Service.cs

```diff
@@ -1,6 +1,11 @@
-﻿using System.Reflection;
+﻿using System;
+using System.Diagnostics.SymbolStore;
```

Comment:
nit: this is unused.

Reply:
Done

Reply:
> I think this is ok... we have an ongoing discussion about .NET version vs. CLR version as runtime, but let's not block this PR because of that. We can reiterate on that.
> 
> I'd suggest not closing #199 and continue the discussion there, plus as agreed this will be also a topic on our next .NET sync.
> 
> Thanks @SergeyKleyman.

I prefer to close #199 as done since this PR does implement what is required in #199. I'll open a follow up to revisit per our discussion here. WDTY?

Reply:
👍

Reply:
I've opened #317 to follow up.